src/pyodide/internal/workers.py

```diff
@@ -416,6 +441,8 @@ def _make_blob_entry(e):
 
 
 def _is_iterable(obj):
+    if isinstance(obj, (str, bytes)):
+        return False
```

Comment:
Yeah, it's unusual to want this. Probably it would have been better if they had been designed from the start to not be iterable but have a method that gives an iterable view.